docker-wrapper.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+# Variables setting
+HOST=`hostname -i`
```

Comment:
Where is it used?

Reply:
In the configuration file:
echo "embedded.http.host=${HOST}" >> /opt/petals-deployer/conf/petals-cli.default